### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

File: Support/hi.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// Messages for Hindi (हिन्दी)
+// Exported from translatewiki.net
+// Author: Abijeet Patro
+
```
Comment (tvision251) [inline Support/hi.lproj/Localizable.strings:4]: `library_refresh_error.retrieve.description` 
`library_refresh_error.parse.description`
`library_refresh_error.process.description`

Comment (tvision251) [reply Support/hi.lproj/Localizable.strings:4]: @translatewiki Please double check all strings for Hindi language.

File: Support/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,99 @@
+// Messages for Italian (italiano)
+// Exported from translatewiki.net
+// Author: Luca.favorido
+
```
Comment (tvision251) [inline Support/it.lproj/Localizable.strings:4]: @translatewiki You missed some strings for Italian.

File: Support/skr-arab.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,109 @@
+// Messages for Saraiki (Arabic script) (سرائیکی)
+// Exported from translatewiki.net
+// Author: Saraiki
+
```
Comment (tvision251) [inline Support/skr-arab.lproj/Localizable.strings:4]: @translatewiki You missed some strings for Arabic.

File: Support/hi.lproj/Localizable.strings (hunk 1 above)
Comment (kelson42) [reply Support/hi.lproj/Localizable.strings:4]: > `library_refresh_error.retrieve.description` 
> `library_refresh_error.parse.description`
> `library_refresh_error.process.description`

what is the problem?

Comment (kelson42) [reply Support/hi.lproj/Localizable.strings:4]: > @translatewiki Please double check all strings for Hindi language.

what is the problem?

**Comment by kelson42** [comment]
@BPerlakiH Can you please do the necessary to review the PR. If something has be fixed upstream in Translatewiki, we need to know as well. The idea is that all PR from Translatewiki can go smooth the `main` branch without manual fixing.

**Comment by BPerlakiH** [comment]
@kelson42  I've pushed my changes here, to fix things.
There were a couple of issues:
- we had 3 empty Spanish translation files in the project already. Only one of them was removed in this PR, therefore it could not build properly.
- there was an export from translate-wiki under the file "qqq" this is not a valid language, it's called "Message Documentation" and should not be imported into the project.
- there were 2 keys/terms which should not be translated at all, as they are only used in "Preview" which is a developer mode, and it won't be part of the final application. These 2 terms were: 
**article_cell.search_result.title** and **article_cell.search_result.snippet.template** one of them was containing "Lorem ipsum..." which was somehow missed earlier. Now these 2 are removed in this PR, but **should also be removed upstream on translatewiki.**

**Review by BPerlakiH — APPROVED** [review]

**Comment by kelson42** [comment]
> * there was an export from translate-wiki under the file "qqq" this is not a valid language, it's called "Message Documentation" and should not be imported into the project. (now also removed).

We can not remove `qqq`. This is indeed the documentation. Can you put a rule to just ignore it? See #561

**Comment by kelson42** [comment]
> > * there was an export from translate-wiki under the file "qqq" this is not a valid language, it's called "Message Documentation" and should not be imported into the project. (now also removed).
> 
> We can not remove `qqq`. This is indeed the documentation. Can you put a rule to just ignore it? See #561

> * there were 2 keys/terms which should not be translated at all, as they are only used in "Preview" which is a developer mode, and it won't be part of the final application. These 2 terms were:
>       **article_cell.search_result.title** and **article_cell.search_result.snippet.template** one of them was containing "Lorem ipsum..." which was somehow missed earlier. Now these 2 are removed in this PR, but **should also be removed upstream on translatewiki.**

TW relies on the master of English, if you remove the strings from the master in English, they won't appear in Translatewiki and won't appear as well in next PR. Can you please do that as well?

**Comment by BPerlakiH** [comment]
> TW relies on the master of English, if you remove the strings from the master in English, they won't appear in Translatewiki and won't appear as well in next PR. Can you please do that as well?

This PR is removing those strings, so after merging this we should be good.

**Pull request merged**